sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Statsbeat/Statsbeat.cs

```diff
@@ -41,6 +45,61 @@ internal static class Statsbeat
 
         internal static MeterProvider s_attachStatsBeatMeterProvider;
 
+        internal static Regex s_endpoint_pattern = new("^https?://(?:www\\.)?([^/.-]+)");
+
+        internal static readonly HashSet<string> EU_Endpoints = new()
+        {
+            "westeurope",
```

Comment:
nit: arrange in alphabetical order, later it would be easy to check and update the list.